subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3128,12 +3128,37 @@ static inline void event_len_prep(struct ll_conn *conn)
 		struct pdu_data_llctrl_length_req *lr;
 		struct pdu_data *pdu_ctrl_tx;
 		struct node_tx *tx;
+		/*
+		 * Using bool instead of u8_t would increase code size
```

Comment:
Please remove this comment. The sample may or may not exist in the future, and code sizes depend on many factors, including redundant come blocks that come and go as the code evolves. If you really want a comment please don't mention a sample in particular, something like:

```
/* Use u8_t instead of bool since it generates smaller code in this case */
```